graph/appender/unused_service.go

```diff
@@ -7,9 +7,25 @@ import (
 
 	"github.com/kiali/kiali/config"
 	"github.com/kiali/kiali/graph/tree"
+	"github.com/kiali/kiali/kubernetes"
 	"github.com/kiali/kiali/log"
 )
 
+type UnusedServiceAppender struct {
+}
+
+func (a UnusedServiceAppender) AppendGraph(trees *[]tree.ServiceNode, namespaceName string) error {
```

Comment:
`(a UnusedServiceAppender)` doesn't seem useful here?

Reply:
Ah, sorry, it's from the interface. Ignore my comment :)

Reply:
Is that not necessary to implements the Interface?